dmutils/user.py

```diff
@@ -63,6 +64,32 @@ def has_role(self, role):
     def has_any_role(self, *roles):
         return any(self.has_role(role) for role in roles)
 
+    def is_part_of_team(self):
+        if self.has_role('buyer'):
+            if self.teams:
+                return True
+        return False
+
+    def is_team_lead(self):
```

Comment:
It's my understanding that a user will only be allowed in one team, however some parts of the code suggest we are anticipating the ability to be in more than one team, e.g. the `return any(t.get('is_team_lead', False) for t in self.teams)` line below. Should `is_team_lead()` therefore accept a param to check whether they are a team lead of a specific team, for future proofing?

Reply:
it will now only return True when all teams have that permission.